### DISCUSSION TITLE
Remove Mesh # Loaded and Buzz / beep when starting up printer

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#412] [Ideas]
When the printer is turned on and you have the Mesh Loaded from any slot, you're greeted with a beep and with "Mesh 0 Loaded" or mesh 1 whatever slot you saved last. 

I find it unnecessary. The only time this should be said or the beep should happen is when you manually select the slot to be loaded, or for whatever other reason, just not when the printer first starts up.

so to stop this....

in FILE --> Marlin/src/module/**settings.cpp**

copy lines 2851 - 2862 (remove `status = true;`)

replace it with

```        
      #if ENABLED(DWIN_LCD_PROUI)
          if (BedLevelTools.meshvalidate()) {

          }
          else {
            status = true;

          }
        #endif
```

take
```    
    #if ENABLED(DWIN_LCD_PROUI)
      if (BedLevelTools.meshvalidate()) {
        ui.status_printf(0, GET_TEXT_F(MSG_MESH_LOADED), bedlevel.storage_slot);
        DONE_BUZZ(true);
      }
      else {
        bedlevel.invalidate();
        LCD_MESSAGE(MSG_UBL_MESH_INVALID);
        DONE_BUZZ(false);
      }
    #endif
```

and paste into **dwin.cpp** ~3618 under `void UBLLoadMesh()` 
like so

```  
  void UBLLoadMesh() {
    if (bedlevel.storage_slot < 0) bedlevel.storage_slot = 0;
    settings.load_mesh(bedlevel.storage_slot);
    #if ENABLED(DWIN_LCD_PROUI)
      if (BedLevelTools.meshvalidate()) {
        ui.status_printf(0, GET_TEXT_F(MSG_MESH_LOADED), bedlevel.storage_slot);
        DONE_BUZZ(true);
      }
      else {
        bedlevel.invalidate();
        LCD_MESSAGE(MSG_UBL_MESH_INVALID);
        DONE_BUZZ(false);
      }
    #endif
  }
```

**Comment by mriscoc** [comment]
It is not an issue, but a personal preference. A good candidate for a PR with a proper enabling LABEL switch

**Comment by jvcjbl** [comment]
This too surprised me with the recent firmware update. Is there a way to disable it?

**Reply by mriscoc** [reply]
That is to give the user a feedback that there is a valid mesh in the selected slot.

**Reply by jvcjbl** [reply]
I understand this. Is there a way to make it quiet? I have my printer in a spot where I do prints really early in the morning when everyone is sleeping and that loud beep was not expected as several other firmwares don't do this behavior loading for slot 0

**Reply by mriscoc** [reply]
In the Advanced Settings menu there is an option to disable sound beeps.
